hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/testutils/BlockDeletingServiceTestImpl.java

```diff
@@ -80,6 +80,7 @@ public void start() {
           break;
         }
         Future<?> future = this.getExecutorService().submit(svc);
+
```

Comment:
This line may not be required to modified and this case this test will remain unchanged.

Reply:
Left over from previous additions to `BlockDeletingServiceTestImpl` - unnecessary and removed.